app/Utils/Process.php

```diff
@@ -7,17 +7,26 @@
 
 class Process extends SymfonyProcess
 {
+
     /**
      * Run the process. If the app is in debug mode, the process will not be executed.
      *
      * @param callable|null $callback
      * @param array $env
      * @return int
      */
-    public function run(?callable $callback = null, array $env = []): int
+    public function run(?callable $callback = null, array $env = [], bool $print_to_log = true): int
     {
         if (config('app.debug') === false || config('commands.run_in_debug') === true) {
-            return parent::run($callback, $env);
+            $return_value = parent::run($callback, $env);
+            if($print_to_log) {
+                if($return_value === 0) {
+                    Log::info("Command: " . $this->getCommandLine() . " executed successfully.");
+                } else {
+                    Log::error("Command: " . $this->getCommandLine() . " failed with error code: " . $return_value . "\nWith output: " . $this->getOutput() . " and error output: " . $this->getErrorOutput());
+                }
+            }
+            return $return_value;
```

Comment:
`$print_to_log` is unnecessarily verbose I think, you could simply call it `$log` or `$verbose`

Reply:
Thank you, updated it